build.cmd

```diff
@@ -1,2 +1,2 @@
-@echo off
-%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -ExecutionPolicy unrestricted -Command "& '%~dp0\tools\psake.ps1' %*"
+dotnet test -c release
```

Comment:
Use dotnet build to restore/test/build then use dotnet build to run a [target](https://github.com/mg-alfa-dev/DeployToAzure/pull/7) for ILMerging the output.

Reply:
A quick mention in the readme would probably be good.